index.html

```diff
@@ -204,10 +205,52 @@ <h3 class="text-small">Graphics</h3>
                     <span class="level">AA+</span>
                   </div>
                   <div class="icons">
-                    <div class="pass">Icon</div>
+                    <div class="pass">
+                      <svg
+                        xmlns="http://www.w3.org/2000/svg"
+                        xmlns:xlink="http://www.w3.org/1999/xlink"
+                        width="15px"
+                        height="100%"
+                        viewBox="0 -4 30 30"
+                        version="1.1"
+                      >
+                        <defs>
+                          <linearGradient
+                            x1="50%"
+                            y1="0%"
+                            x2="50%"
+                            y2="100%"
+                            id="linearGradient-1"
+                          >
+                            <stop stop-color="#1DD47F" offset="0%" />
+                            <stop stop-color="#0DA949" offset="100%" />
+                          </linearGradient>
+                        </defs>
+                        <g
+                          stroke="none"
+                          stroke-width="1"
+                          fill="none"
+                          fill-rule="evenodd"
+                        >
+                          <g
+                            id="ui-gambling-website-lined-icnos-casinoshunter"
+                            transform="translate(-735.000000, -1911.000000)"
+                            fill="url(#linearGradient-1)"
+                            fill-rule="nonzero"
+                          >
+                            <g transform="translate(50.000000, 1871.000000)">
+                              <path
+                                d="M714.442949,40.6265241 C715.185684,41.4224314 715.185684,42.6860985 714.442949,43.4820059 L697.746773,61.3734759 C697.314529,61.8366655 696.704235,62.0580167 696.097259,61.9870953 C695.539848,62.0082805 694.995328,61.7852625 694.600813,61.3625035 L685.557051,51.6712906 C684.814316,50.8753832 684.814316,49.6117161 685.557051,48.8158087 C686.336607,47.9804433 687.631056,47.9804433 688.410591,48.8157854 L696.178719,57.1395081 L711.589388,40.6265241 C712.368944,39.7911586 713.663393,39.7911586 714.442949,40.6265241 Z"
+                              />
+                            </g>
+                          </g>
+                        </g>
+                      </svg>
+                    </div>
                   </div>
                 </div>
               </div>
+            <div>
```

Comment:
You accidentally added an opening `div` tag here. So, please make sure to correct that.